mappers/bluetooth_mapper/configuration/config.yaml

```diff
@@ -0,0 +1,76 @@
+mqtt:
```

Comment:
Please mention that this is just an example and how it can be modified in 2-3 lines of comment above.

Reply:
perhaps we need not fill any values and ask user to fill them according to their device. We can add a small readme here for helping write config files

Reply:
done thanks